docs/source/scripts.rst

```diff
@@ -34,4 +34,15 @@ ivadomed_extract_small_dataset
 
 .. autofunction:: ivadomed.scripts.extract_small_dataset.extract_small_dataset
 
+ivadomed_training_curve
+""""""""""""""""""""""""""""""
```

Comment:
needs to be the same number of chars as the title

Reply:
done in 384de6c